src/components/nav-bar/component.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Link from 'redux-first-router-link';
+import Dropdown from 'react-dropdown';
+
+import { menuItems } from './constants';
+
+import './styles.scss';
+
+const NavBar = ({ setRouter }) => {
+  const handleChange = (item) => setRouter({ type: item.type, pathname: item.path })
```

Comment:
Are you sure this is working and setting the subtab? Visually it's impossible to say